template-parts/header/header-search.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Displays the search form for the header.
+ *
+ * @package Newspack
+ */
+?>
+
+<div class="header-search-contain">
+	<button id="search-toggle" on="tap:AMP.setState({searchVisible: !searchVisible})" aria-controls="search-menu" [aria-expanded]="searchVisible ? 'true' : 'false'" aria-expanded="false">
```

Comment:
Unless you're aware of a reason why this might be an AMP problem, let's space this out according to WordPress coding convention:

```suggestion
	<button id="search-toggle" on="tap:AMP.setState( { searchVisible: ! searchVisible } )" aria-controls="search-menu" [aria-expanded]="searchVisible ? 'true' : 'false'" aria-expanded="false">
```

Reply:
Updated - it doesn't seem to cause issues!